guides/common/modules/ref_required-red-hat-repositories.adoc

```diff
@@ -0,0 +1,37 @@
+[id="required-red-hat-repositories"]
+= Required Red Hat repositories
+
+You need the following repositories to manage hosts with {RHEL}.
+
+For {RHEL} 10 hosts::
+* *{RHEL} 10 for x86_64 - BaseOS (RPMs)*
```

Comment:
I realize this is a little late because your PR has been merged. I was looking through this section in the Pantheon preview build and found the bold font hard to read. Is there any reason not to use regular font for the repos?